src/database/queries/applicationFormFields/updateById.sql

```diff
@@ -0,0 +1,6 @@
+UPDATE application_form_fields
+SET
+	label = coalesce(:label, label),
+	instructions = coalesce(:instructions, instructions)
```

Comment:
The label and instructions are the only nullable fields but `coalesce` means we cannot set them to null. I think it's OK because we can use an empty string as a workaround in the rare case where we want to go from a string of some kind to null.

Reply:
Oh snap that's awkward... what do we even do 😭

(I'll see if I can think of an elegant solution)